Field_Simulation/Simulation/WindowsFormsApp1/Form1.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using System.Threading;
+
+namespace WindowsFormsApp1
+{
+    public partial class Form1 : Form
+    {
+        Mutex cleanM = new Mutex();
```

Comment:
I prefer `lock`ing a specific object rather than using a `Mutex` which is completely unnecessary (we don't really work cross-os)

Reply:
Use access modifiers to emphasize fields' accessibility level.